# Update these values based on hexapod.urdf.xacro or other way around
base:
  radius: 0.08 # meters
  height: 0.04 # meters
  z_offset: 0.01 # meters
coxa_length: 0.01 # meters
femur_length: 0.04 # meters
tibia_length: 0.08 # meters

foot:
  radius: 0.165 # TODO: remove this so it's no longer hard-coded
leg_L1:
  hip_angle: 120 # degrees
leg_R1:
  hip_angle: 60 # degrees
leg_L2:
  hip_angle: 180 # degrees
leg_R2:
  hip_angle: 0 # degrees
leg_L3:
  hip_angle: 240 # degrees
leg_R3:
  hip_angle: 300 # degrees